misc/systemd/ipfs.service

```diff
@@ -1,9 +1,29 @@
 [Unit]
-Description=IPFS Daemon
+Description=InterPlanetary File System (IPFS) daemon
+After=network.target
```

Comment:
Networking isn't strictly speaking necessary. Other services should be able to load content already available on the local node offline.

Reply:
But the default config does bind sockets, which will fail without a network target already present, right?

Reply:
Hm, maybe? We usually bind to 0.0.0.0 (no networking required) and 127.0.0.1. I assumed systemd brought up the loopback interface before starting services.

However, I guess it doesn't hurt to enable this for now. My concern is that I'd like early units to be able to pull files out of IPFS if necessary.

Reply:
network.target just means that the network stack is started, not necessarily that there is any network connection possible, this would be network-online.target.

I think setting network.target is a sane default without blocking any requests on boot.